<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgChangePCValidity</class>
 <widget class="QDialog" name="dlgChangePCValidity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbIncrease">
     <property name="title">
      <string> Increase validity ... </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbOneDay">
        <property name="text">
         <string>with one day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOneWeek">
        <property name="text">
         <string>with one week</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTwoWeek">
        <property name="text">
         <string>with two weeks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOneMonth">
        <property name="text">
         <string>with one month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbThreeMonths">
        <property name="text">
         <string>with three months</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbHalfYear">
        <property name="text">
         <string>with half year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCustom">
        <property name="text">
         <string>to custom date</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDateEdit" name="deCustom">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="date">
           <date>
            <year>2013</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>248</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbOneDay</tabstop>
  <tabstop>rbOneWeek</tabstop>
  <tabstop>rbTwoWeek</tabstop>
  <tabstop>rbOneMonth</tabstop>
  <tabstop>rbThreeMonths</tabstop>
  <tabstop>rbHalfYear</tabstop>
  <tabstop>rbCustom</tabstop>
  <tabstop>deCustom</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
